projects/ubuntu-20.04/ubuntu-20.04-mysql-8.0/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+
+services:
+  catalog:
+    image: mysql:8.0.29
+    environment:
+      - MYSQL_ROOT_PASSWORD=testpassword
+      - MYSQL_USER=irods
+      - MYSQL_PASSWORD=testpassword
+    # MySQL 8.0 defaults to using UTF-8 (utf8mb4/utf8mb4_0900_ai_ci). iRODS requires that queries
+    # be case-sensitive. Therefore, we set the --collation-server option to one that satisfies that
+    # requirement.
+    #
+    # --disable-log-bin is needed so that setup succeeds. Without this option, the following
+    # error is produced and setup fails:
+    #
+    #    ERROR 1418 (HY000): This function has none of DETERMINISTIC, NO SQL,
+    #    or READS SQL DATA in its declaration and binary logging is enabled
+    #    (you *might* want to use the less safe log_bin_trust_function_creators
+    #    variable)
+    #
+    command: "--transaction-isolation=READ-COMMITTED --collation-server=utf8mb4_0900_as_cs --disable-log-bin"
+    security_opt:
+      - seccomp:unconfined
```

Comment:
Here is the issue that lead me to add that line. Without it, we can't see log output.

https://github.com/docker-library/mysql/issues/303

Removing the option doesn't appear to harm its operation. We just can't see any errors logged by the MySQL container and we see `mbind: Operation not permitted` show up for every icommand invocation.

We can leave the option there, but commented out in case someone needs to enable the MySQL logs. We can also add documentation to the README about it.

Thoughts?

Reply:
Ah, okay. Thanks for the explanation. I think we can leave it, then, let's just add a comment to the effect of what you've already said here so that we know why the security options have been disabled. This is the testing environment, after all, so it is what it is.

Reply:
I fully agree.